streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -293,7 +294,18 @@ public void commitOffsets() {
                 consumedOffsetsAndMetadata.put(partition, new OffsetAndMetadata(offset));
                 stateMgr.putOffsetLimit(partition, offset);
             }
-            consumer.commitSync(consumedOffsetsAndMetadata);
+            try {
+                consumer.commitSync(consumedOffsetsAndMetadata);
+            } catch (final CommitFailedException cfe) {
+                for (final Map.Entry<TopicPartition, OffsetAndMetadata> topicPartitionOffsetAndMetadataEntry : consumedOffsetsAndMetadata.entrySet()) {
+                    log.warn("FAILED COMMIT: topic={} partition={} offset={}",
```

Comment:
Could we have one warning log entry instead of multiple lines for a single exception? It will help with log file greps / etc I think.

nit: `TopicPartition` / `OffsetsAndMetadata` classes have their own `toString` function that can be used, so we just need to use that, so printing the map itself should be fine.